pkg/querier/querier.go

```diff
@@ -43,6 +43,7 @@ type Config struct {
 	ExtraQueryDelay          time.Duration    `yaml:"extra_query_delay,omitempty"`
 	IngesterMaxQueryLookback time.Duration    `yaml:"query_ingesters_within,omitempty"`
 	Engine                   logql.EngineOpts `yaml:"engine,omitempty"`
+	MaxConcurrent            int              `yaml:"max_concurrent"`
```

Comment:
this is the only configuration which is used in `frontend` worker. I have modified the code accordingly herehttps://github.com/grafana/loki/blob/2f3632878fbf31e31f2d2a998288d1eb675cb479/pkg/loki/modules.go#L160

Hope this change is fine